File: unicardsoftwaredevelopment-androidmobileappfortopupandrefund-c0e3674d02c4/app/src/main/java/au/com/unicard/tafenswwallet/ui/refund/RefundFragment.java
package au.com.unicard.tafenswwallet.ui.refund;

import androidx.lifecycle.Observer;
import androidx.lifecycle.ViewModelProviders;

import android.os.Bundle;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import au.com.unicard.tafenswwallet.R;

public class RefundFragment extends Fragment {

    private RefundViewModel refundViewModel;

    public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup container,
                             @Nullable Bundle savedInstanceState) {

        refundViewModel = ViewModelProviders.of(this).get(RefundViewModel.class);
        View root = inflater.inflate(R.layout.fragment_refund, container, false);
        final TextView textView = root.findViewById(R.id.text_refund);

        refundViewModel.getText().observe(this, new Observer<String>() {
            @Override
            public void onChanged(@Nullable String s) {
                textView.setText(s);
            }
        });
        return root;
    }
}

File: TAFEWallet/app/src/main/java/com/yuan/tafewallet/ui/refund/RefundFragment.kt
package com.yuan.tafewallet.ui.refund

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.TextView
import androidx.fragment.app.Fragment
import androidx.lifecycle.Observer
import androidx.lifecycle.ViewModelProviders
import com.yuan.tafewallet.R

class RefundFragment : Fragment() {

    private lateinit var refundViewModel: RefundViewModel

    override fun onCreateView(
        inflater: LayoutInflater,
        container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        refundViewModel =
            ViewModelProviders.of(this).get(RefundViewModel::class.java)
        val root = inflater.inflate(R.layout.fragment_refund, container, false)
        val textView: TextView = root.findViewById(R.id.text_refund)
        refundViewModel.text.observe(this, Observer {
            textView.text = it
        })
        return root
    }
}